components/Map/Map.jsx

```diff
@@ -478,7 +478,6 @@ class Map extends React.Component {
     dispatchUpdateNcId(null);
 
     // Reset councilSelector.
-    dispatchUpdateSelectedCouncils([]);
```

Comment:
Since we're removing this, can we also remove `dispatchUpdateSelectedCouncils` initialization on line 472?